src/templates/_analytics.ejs

```diff
@@ -0,0 +1,12 @@
+<script async src='https://www.google-analytics.com/analytics.js'></script>
+    <script>
```

Comment:
Indent nit

Reply:
actually I'm doing it for a reason

the CSP hash we're using cares about indentation, and the `<%- include ... %>` tag in ejs only indents the first line to level it's in, the other lines are unindented... so I'm doing this to make it pass CSP :(((

(one solution might be to switch from hashes to nonces)